docs/release_notes/v0.20.0-beta.md

```diff
@@ -0,0 +1,48 @@
+# Spice v0.20.0 (Nov 04, 2024)
+
+Spice v0.20.0-beta ...
+
+## Highlights in v0.20.0
+
+- support metal and cuda accelerators
+- promote postgres connector to RC quality
+- improve localpod acceleration
+- spice upgrade now upgrades the runtime
+
+## Breaking changes
+
+- ODBC profile change
+
```

Comment:
DuckDB and SQLite changes requiring database files within the working directory, e.g. `sqlite_file` etc